package.json

```diff
@@ -70,5 +70,8 @@
     "rollup": "4.23.0",
     "typescript": "5.6.3",
     "yorkie": "2.0.0"
+  },
+  "engines": {
+    "node": "^16.20.0 || ^18.0.0 || ^20.0.0 || ^22.0.0"
```

Comment:
I generally consider changing explicitly supported node versions as a breaking change. I don't mind doing it, but I do think we should bump major version for clarity. WDYT?

Reply:
It's handled via release-please, so if we update the title of this PR to be `"feat!: Set Node.js support"` (or some such thing with "feat!" at the front), when this is merged, release-please creates a major version bump.

Reply:
great, right. I probably should have checked 😄